addons/auth_passkey_portal/static/tests/test_passkey_portal_create.js

```diff
@@ -0,0 +1,93 @@
+import { registry } from "@web/core/registry";
```

Comment:
I think in the guidelines test tours should be in `/static/tests/tours`, not just `/static/tests`.

And you can add the 3 tours on the same file, as they are closely related.